AutoRest/Modelers/Swagger.Tests/SwaggerModelerTests.cs

```diff
@@ -480,6 +480,20 @@ public void TestConstants()
             Assert.Equal(false, clientModel.ModelTypes.First(m => m.Name == "Error").ContainsConstantProperties);
         }
 
+        //[Fact]
```

Comment:
This is commented because I'm not familiar with the C# test framework, and was unable to get the test suite to run locally. I've included a swagger file that reproduces the issue (a cut down version of the network spec), and included this commented out test stub of what I imagine the test case might look like. I'm hoping you might be able to run the test case and adjust if necessary....